Tests/Pcap++Test/TestDefinition.h

```diff
@@ -35,6 +35,7 @@ PTF_TEST_CASE(TestPcapFilters_matchPacketWithFilter_static);
 PTF_TEST_CASE(TestPcapFiltersLive);
 PTF_TEST_CASE(TestPcapFilters_General_BPFStr);
 PTF_TEST_CASE(TestPcapFiltersOffline);
+PTF_TEST_CASE(TestPcapFilters_matchPacketWithFilter_DifferentInterfaces);
```

Comment:
Please change the test name to fit 30 chars or less

Reply:
Done.